packages/bpk-component-spinner/src/BpkExtraLargeSpinner.js

```diff
@@ -26,19 +28,22 @@ import SPINNER_TYPES from './spinnerTypes';
 
 const getClassName = cssModules(STYLES);
 
-const BpkExtraLargeSpinner = props => {
+type Props = {
+  type: $Keys<typeof SPINNER_TYPES>,
+  className: ?string,
+};
+
+const BpkExtraLargeSpinner = (props: Props) => {
   const { type, className, ...rest } = props;
-  const classNames = [
+  const classNames = getClassName(
     'bpk-spinner',
     'bpk-spinner--extra-large',
     `bpk-spinner--${type}`,
-  ].map(getClassName);
-
-  if (className) {
-    classNames.push(className);
-  }
+    className && className,
```

Comment:
```suggestion
    className,
```

You don't need to do the `className && className` here, just `className` works

I'm getting deja vu as I'm sure I left this exact comment on another PR of yours about a month ago 😛

Reply:
Yeah my ide was throwing errors when I did this yesterday so had to put that in for now

Reply:
Are you sure? We use this in many different components so it would be unusual for it to only error on these instances and not all of them, if it's really broken

Reply:
As I said was just my ide yesterday was planning on fixing it this morning after a pc restart

Reply:
Done